lib/foreman_theme_satellite/documentation.rb

```diff
@@ -80,5 +80,11 @@ module Documentation
     SPECIAL_LINKS = [
       [/docs\.theforeman\.org\/.*?\/Managing_Hosts\/.*?registering-a-host.*?managing-hosts/i, "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts"],
     ]
+
+    DOCS_GUIDES_LINKS = {
+      'Managing_Hosts' => {
+        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts",
```

Comment:
Where does the specific anchor come from?

Reply:
https://github.com/theforeman/foreman/blob/0b40a64be53fb54843bccf2aa193cab0632c83b3/webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js#L182-L183

Reply:
But that doesn't have an anchor, only a guide name and a chapter. Yet the branded URL has `Registering_Hosts_managing-hosts` as the anchor.

Reply:
Ah, you mean the target URL. I copied that from  SPECIAL_LINKS, which is what was used to brand this link before it got converted to the new docs-handler.